illumio_connector.py

```diff
@@ -478,24 +476,17 @@ def _handle_create_enforcement_boundary(self, param):
     def _handle_get_workloads(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
 
-        enforcement_mode = (
-            param.get("enforcement_mode").lower()
-            if param.get("enforcement_mode")
-            else None
-        )
-        connectivity = (
-            param.get("connectivity").lower() if param.get("connectivity") else None
-        )
+        enforcement_mode = param.get("enforcement_mode").lower()
```

Comment:
The call to lower() here will break if enforcement_mode is None, no?

Reply:
Made appropriate changes to overcome the issue.